Dockerfile

```diff
@@ -19,6 +19,3 @@ RUN pip install --editable .
 RUN useradd worker
 RUN chown worker:worker -R /home/worker
 USER worker
-
-ENTRYPOINT ["chanjo", "report"]
```

Comment:
I've removed this since as it is it makes it impossible to run the command. This is how the command is runned:

chanjo -d database_uri <command>

An option would be to leave just "chanjo" inside the entrypoint